steps/versionbox-deploy/0.0.1/step.yml

```diff
@@ -0,0 +1,56 @@
+title: versionbox-deploy
+summary: |
+  Send your generated applications to VersionBox.
+description: "Send your generated ipa/apk files to VersionBox, automatically upload
```

Comment:
Please include the link to VersionBox, to let the users understand what this step does.